week2/extra-homework/counters.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const fs = require('fs');
+
+class Counters {
+  constructor(file) {
+    this.file = file;
+  }
+
+  static read(file, encoding) {
```

Comment:
The encoding is mostly `utf8` in your case since it's counters `json` file on any write or in any case you have.

Reply:
That is right, Hard coded the `fs.readFile` to `utf8`